Clockwise.Redis.Tests/CircuitBreakerStorageTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Threading.Tasks;
+using FluentAssertions;
+using StackExchange.Redis;
+using Xunit;
+
+namespace Clockwise.Redis.Tests
+{
+    public class CircuitBreakerStorageTests : IDisposable
+    {
+        [Fact]
+        public async Task SingalingState()
```

Comment:
This test should be named to something more informative and probably split into a few different tests to verify the different state transitions.